test/microsoftdynamicscrm/metadata/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/amp-labs/connectors/common"
+	msTest "github.com/amp-labs/connectors/test/microsoftdynamicscrm"
+	"github.com/amp-labs/connectors/test/utils"
+)
+
+var (
+	objectName       = "account"
+	objectNamePlural = "accounts"
+)
+
+// we want to compare fields returned by read and schema properties provided by metadata methods
+// they must match for all such objects
+func main() {
```

Comment:
Algorithm:
* Select object (ex: account)
* Ask for Metadata
* Query endpoint via READ
* Response filed must match schema acquired from Metadata